src/uu/stdbuf/src/stdbuf.rs

```diff
@@ -206,18 +203,15 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
             return match e.kind() {
                 std::io::ErrorKind::PermissionDenied => Err(USimpleError::new(
                     126,
-                    get_message("stdbuf-error-permission-denied"),
+                    translate!("stdbuf-error-permission-denied", "command" => command_name.quote()),
                 )),
                 std::io::ErrorKind::NotFound => Err(USimpleError::new(
                     127,
-                    get_message("stdbuf-error-no-such-file"),
+                    translate!("stdbuf-error-no-such-file", "command" => command_name.quote()),
```

Comment:
Are the "command" arguments intentional? The ftl files don't expect such arguments.